DownloadTask.py

```diff
@@ -4,7 +4,7 @@
 
 
 class DownloadTask:
-    def __init__(self, callback, hash, size, name, category):
+    def __init__(self, callback, hash, size, name, category, dldir, dlext, dlsize, dlnzbtomedia):

```

Comment:
What is the purpose of these params? The fact that it should be forwarded to nzbtomedia would be based on the category, no?